webpack.base.config.js

```diff
@@ -12,9 +14,22 @@ const UI_ROOT = path.resolve(__dirname, 'src/ui/');
 const STATIC_ROOT = path.resolve(__dirname, 'static/');
 const DIST_ROOT = path.resolve(__dirname, 'dist/');
 
+console.log(ifProduction('production', 'development'))
```

Comment:
Don't need

Reply:
It's useful to see what's actively building